Teapot/Teapot.swift

```diff
@@ -274,7 +276,7 @@ open class Teapot {
 
         if let headerFields = headerFields {
             for headerField in headerFields {
-                if headerField.key.lowercased() == "content-type" {
+                if headerField.key == "Content-Type" {
```

Comment:
I would strongly recommend leaving this - that way if someone does `Content-type` we still catch the fact that it has a content type

Reply:
I think I accidentally reverted this when merging back from master. 🙈